packages/template-retail-react-app/app/marketing-cloud-email-link.js

```diff
@@ -0,0 +1,105 @@
+/*
```

Comment:
Let's put these under the `util` folder or create a new directory for these like `marketing-cloud`